src/app/components/Account/ShowMoreTokensLink.tsx

```diff
@@ -37,7 +37,7 @@ export const ShowMoreTokensLink: FC<ShowMoreTokensLinkProps> = ({ account, token
   }
 
   return (
-    <StyledLink to={erc20link} color="inherit">
+    <StyledLink to={erc20link} color="inherit" replace>
```

Comment:
Why manual replace?
- https://cd976a58.oasis-explorer.pages.dev/search?q=0x404E70A162487c9Af8982a89a5453f389d5257b1
- Click "+ 1 more"
- Click back

Reply:
ahh yeah that works only in Account details context. 
Moreover, option 3 `Manual control of replace prop` would be much more messy. I did not think about tokens.